content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/carousel.html

```diff
@@ -27,12 +27,15 @@
      data-cmp-autopause-disabled="${carousel.autopauseDisabled}"
      data-cmp-data-layer="${carousel.dataLayerString}">
     <div data-sly-test="${carousel.items && carousel.items.size > 0}"
-         class="cmp-carousel__content">
+         class="cmp-carousel__content"
+         aria-atomic="false"
+         aria-live="${carousel.autoplay ? 'off' : 'polite'}">
```

Comment:
These should be added as a different commit, linked to the proper issue/PR.

Reply:
see commit https://github.com/adobe/aem-core-wcm-components/pull/982/commits/8641772d95bf18884989ee06492ac0f281bbcfd5